### DISCUSSION TITLE
QCalculator - A Vue Component to handle on-screen calculator inputs and present calculated answers

### DISCUSSION THREAD

**Discussion by Qoyyuum** [quasarframework/quasar#15348] [General - Components / Directives / etc]
I can think of a couple of apps that can use this component like a Budgeting app or a Point of Sale app. It's not just an app that adds numbers to a number input field, but it does the calculations and returns the calculated number into the input model field. I can't think of what props/slots that it could use.

Anyone could make it (it's probably done a bunch of times) but would anyone object for this component to be added to the Quasar Framework?

**Comment by brianoflondon** [comment]
Did you ever build something like this?
